SteamBattleBot/Structures/PlayerStructure.cs

```diff
@@ -173,22 +175,22 @@ public void state(SteamFriends.FriendMsgCallback callback, SteamFriends steamFri
             if (enemy.classRandom == 1)
             {
                 if (!shopMode)
-                    steamFriends.SendChatMessage(callback.Sender, EChatEntryType.ChatMsg, String.Format("\nCurrent Status\nYou:\nHP: {0}\nCoins: {1}\nPoints: {3}\nSteam Bot:\nHP: {2}", hp, coins, enemy.hp, points));
+                    steamFriends.SendChatMessage(callback.Sender, EChatEntryType.ChatMsg, String.Format("\nCurrent Status\nYou:\nHP: {0}\nLevel: {4}\nXP: {5}\nCoins: {1}\nPoints: {3}\nSteam Bot:\nHP: {2}", hp, coins, enemy.hp, points, level, exp));
```

Comment:
When you do the {(number)} you need to go from 0-(amount of vars minus one). So it would be {0} {1} {2} etc, having it like {0} {7} {1} etc will cause errors and make it more complex then it needs to be.